jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py

```diff
@@ -2,13 +2,28 @@
 
 from jdaviz.core.registries import tray_registry
 from jdaviz.core.template_mixin import PluginTemplateMixin, DatasetSelectMixin
+from jdaviz.core.user_api import PluginUserApi
 from jdaviz.utils import PRIHDR_KEY, COMMENTCARD_KEY
 
 __all__ = ['MetadataViewer']
 
 
 @tray_registry('g-metadata-viewer', label="Metadata Viewer")
 class MetadataViewer(PluginTemplateMixin, DatasetSelectMixin):
+    """
+    See the :ref:`Metadata Viewer Plugin Documentation <imviz_metadata-viewer>` for more details.
+
+    Only the following attributes and methods are available through the
+    :ref:`public plugin API <plugin-apis>`:
+
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.show`
+    * :meth:`~jdaviz.core.template_mixin.PluginTemplateMixin.open_in_tray`
+    * ``dataset`` (:class:`~jdaviz.core.template_mixin.DatasetSelect`):

```

Comment:
Hmm. Should we change this to "data_label" to be consistent with another plugin?

Reply:
I don't think so.  In this case, it is a selectable dropdown, as opposed to a read-only indication of the top-layer.  If anything, I'd be in favor of changing other instances of `data_label` to `dataset` and have it just show as a single choice.  That then gives us flexibility to change it to a proper dropdown in the future.

Reply:
OK, let's leave it then.